README.md

```diff
@@ -1 +1,77 @@
 # homebrew-tap
+
+Easily install the [Databricks CLI](https://github.com/databricks/cli) using [Homebrew](https://brew.sh/).
+
+Full documentation about installation can be found at:
+* (AWS) https://docs.databricks.com/en/dev-tools/cli/install.html
+* (Azure) https://learn.microsoft.com/en-us/azure/databricks/dev-tools/cli/install
+* (GCP) https://docs.gcp.databricks.com/en/dev-tools/cli/install.html
+
+## Usage
+
+Initialize tap:
+
+```bash
+brew tap databricks/tap
+```
+
+To install or upgrade the Databricks CLI, run:
+
+```bash
+brew install databricks
+```
+
+If this tap has been updated but you don't see it show up locally, you can force an update with:
+
+```bash
+brew update --force
+```
+
+## Installing an older version
+
+If you need to install an older version of the Databricks CLI, you can do so with:
+
+```bash
+brew tap-new databricks/tap-old
```

Comment:
Is this a normal SOP for Homebrew tap users? It seems very cumbersome.

Reply:
Yes, it is not possible to extract into the same tap where it is defined:

```
% brew extract --version=0.218.0 databricks/tap/databricks databricks/tap
Error: Cannot extract formula to the same tap!
```

Reply:
And this is really a workaround -- Homebrew has first-class support for multiple versions of the same tap as long as you keep all versions as a separate file in the "Formula" directory. I don't think that's worth it for us (yet?).